src/components/login/loginContext.js

```diff
@@ -65,6 +72,19 @@ export const LoginProvider = ({ children }) => {
     }
   };
 
+  /** Hashes provided password
+   *
+   * @param {string} password
+   * @returns {string | Error}
+   */
+  const hashPassword = async password => {
```

Comment:
We're still doing the hashing serverside?

Reply:
What I decided to do was hash it client side, then rehash again and then salt. You can never hash a password too many times.

Reply:
Beautiful 🔐🔐🔐